src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Component.targets

```diff
@@ -180,6 +180,10 @@ Copyright (c) .NET Foundation. All rights reserved.
       <UseSharedCompilation>true</UseSharedCompilation>
     </PropertyGroup>
 
+    <PropertyGroup>
+      <RazorComponentErrorLog Condition="'$(RazorComponentErrorLog)'==''"><$(IntermediateOutputPath)$(RazorTargetName).Component.log/RazorComponentErrorLog>
```

Comment:
O_o

Reply:
Can we make this `$(RazorTargetName).TempCompilation.log`

Reply:
yeah, this is teh error.